SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+DarkReader is known to have no intentional security vulnerabilities.
+If reported, vulnerabilites will be fixed in the shortest time possible,
+with minimal impact possible.
+
+## Supported Versions
+
+Latest version is supported.
+
+| Version   | Supported          |
+| --------- | ------------------ |
+| 4.9.16    | :white_check_mark: |
+| < 4.9.16  | :x:                |
+
+## Reporting a Vulnerability
+
+In case there is a vulnerability left behind, immediately notify
+to <darkreaderapp@gmail.com>; unless you can fix it yourself,
```

Comment:
> unless you can fix it yourself,

So that everyone can see the PR and abuse the bug right away?

Reply:
I left that bit intentionally, to enable a copyleft relicense, which require the changes the published as soon as possible.